description/description.rdfs.ttl

```diff
@@ -517,6 +599,48 @@ schema:transcript a rdf:Property ;
     rdfs:comment "Als dit MediaObject een AudioObject of VideoObject is, de transcriptie van dat object."@nl ;
     rdfs:comment "Si cet MediaObject est un AudioObject ou VideoObject, la transcription de cet objet."@fr .
 
+schema:unitCode a rdf:Property ;
+    rdfs:domain schema:QuantitativeValue ;
+    rdfs:range xsd:string ;
+    rdfs:label "unit code"@en ;
+    rdfs:label "eenheid code"@nl ;
+    rdfs:label "code d'unité"@fr ;
+    # Hier ook URL naar lijst met codes toevoegen?
+    rdfs:comment "The unit of measurement given using the UN/CEFACT Common Code (3 characters)."@en ;
+    rdfs:comment "De eenheid van de meting, gegeven met behulp van de UN/CEFACT Common Code (3 karakters)."@nl ;
+    rdfs:comment "L'unité de mesure donnée en utilisant le code commun UN/CEFACT (3 caractères)."@fr .
+
+schema:unitText a rdf:Property ;
+    rdfs:domain schema:QuantitativeValue ;
+    rdfs:range xsd:string ;
+    rdfs:label "unit text"@en ;
+    rdfs:label "eenheid tekst"@nl ;
+    rdfs:label "texte d'unité"@fr ;
+    rdfs:comment "A string or text indicating the unit of measurement. Useful if you cannot provide a standard unit code for unitCode."@en ;
+    rdfs:comment "Een string of tekst die de eenheid van de meting aangeeft. Handig als je geen standaard eenheidcode kunt geven voor unitCode."@nl ;
+    rdfs:comment "Une chaîne de caractères ou un texte indiquant l'unité de mesure. Utile si vous ne pouvez pas fournir de code d'unité standard pour unitCode."@fr .
+
+schema:value a rdf:Property ;
+    rdfs:domain schema:QuantitativeValue ;
+    # Integer volgens developer documentation. In de praktijk ook floats?
```

Comment:
Datatype in de SIP moet niet identiek zijn, want daar vragen we specifiek achter kunstwerk afmetingen in millimeters. Voor de KG is xsd:float of xsd:decimal correct